### PR TITLE
feat(tier4_perception_component): refactored launch options

### PR DESCRIPTION
**Pull request by miursh**
## Description

This PR includes the following changes:

- Added the args from perception.launch.xml to tier4_perception_component.launch.xml.
- Modified the args to accept external parameters for better flexibility and configurability.
- Changed the default number of cameras for the traffic light recognition to 2.

These changes enable easier parameterization and customization of the launch configurations from external sources and improve the default setup for signal recognition.
## Tests performed
- Verified in my local environment that the perception module launches without changes in behavior.
- Confirmed that nodes can be controlled using parameters provided externally. (e.g. `use_traffic_light_recognition:=false`)
- Ensured that the TLR works correctly even when there is only one image topic.

## Effects on system behavior

No effects

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: autoware_launch/launch/components/tier4_perception_component.launch.xml [hunk 1]
```diff
@@ -1,31 +1,53 @@
...
     <arg name="objects_filter_method" value="$(var detected_objects_filter_method)"/>
     <arg name="objects_validation_method" value="$(var detected_objects_validation_method)"/>
     <arg name="data_path" value="$(var data_path)"/>
-    <arg name="use_low_height_cropbox" value="false"/>
 
-    <!-- object recognition -->
+    <arg name="use_traffic_light_recognition" value="$(var use_traffic_light_recognition)"/>
+    <arg name="traffic_light_recognition/enable_fine_detection" value="$(var traffic_light_recognition/enable_fine_detection)"/>
+    <arg name="traffic_light_recognition/fusion_only" value="false"/>
+    <arg name="traffic_light_image_number" value="2"/>
```
Comment (kminoda) [inline autoware_launch/launch/components/tier4_perception_component.launch.xml:48]: Any particular reason why the default value is changed from 1 to 2?

**Review by kminoda — COMMENTED** [review]
Thanks :pray:

**Review by kminoda — APPROVED** [review]
LGTM
(Confirmed that awf/autoware launches without failure and at least object_recognition/objects are published with centerpoint)

**Review by technolojin — APPROVED** [review]
LGTM!

**Pull request merged**